product-mini/platforms/linux-sgx/enclave-sample/Enclave/Enclave.cpp

```diff
@@ -377,17 +377,49 @@ handle_cmd_instantiate_module(uint64 *args, uint32 argc)
 
     bh_assert(argc == 5);
 
+    *(void **)args_org = NULL;
+
     if (!runtime_inited) {
-        *(void **)args_org = NULL;
         return;
     }
 
+#if WASM_ENABLE_MULTI_MODULE == 0
+    {
+        int32_t import_count =
+            wasm_runtime_get_import_count(enclave_module->module);
+        WASMExternInstance *imports = NULL;
+
+#if WASM_ENABLE_SPEC_TEST != 0 || WASM_ENABLE_WASI_TEST != 0 \
+    || WASM_ENABLE_LIBC_BUILTIN != 0
+        imports =
+            wasm_runtime_create_imports_with_builtin(enclave_module->module);
```

Comment:
it may allocate memory, should check whether imports is NULL and return if it is NULL?